README.md

```diff
@@ -23,7 +23,7 @@ https://developer.jboss.org/wiki/PerformanceGatesForAcceptingPerformanceFixesInN
 
 Requirements
 ------------
-To build this project you will need a JDK (Java Development Kit) with a minimum version of 11.
+To build this project you will need a JDK (Java Development Kit) with a minimum version of 11 and a Maven with a minimum version of 3.8.2.
```

Comment:
Or use the maven wrapper script bundled with this repo

Reply:
> Or use the maven wrapper script bundled with this repo

agreed and updated the changes. Thanks